datasets/nrel-pds-wtk.yaml

```diff
@@ -99,6 +99,12 @@ Resources:
     Type: S3 Bucket
     Explore:
     - '[Browse Dataset](https://data.openei.org/s3_viewer?bucket=nrel-pds-wtk&prefix=maine%2F)'
+  - Description: Gulf of Mexico wind resource data (2000-2020) in [HDF5](https://portal.hdfgroup.org/display/HDF5) format
+    ARN: arn:aws:s3:::nrel-pds-wtk/gulf_of_mexico/

```

Comment:
Thanks @mheine3000, just a heads up this prefix is showing no data. Once that's resolved, we should be good to merge.

Reply:
I see some data now so should be good to go. You should just be able to "merge latest from base branch" or similar down below this where it is telling you it is out of date. Assuming tests pass after that, I can go ahead and merge into main.

Reply:
All done!